### PR TITLE
[T4A4][T09-A4]Cao Wei

### PR DESCRIPTION
**Pull request by LoneGunman001**

### CODE REVIEW

**Comment by LoneGunman001** [comment]
Ready for review

**Comment by LoneGunman001** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -6,11 +6,12 @@
  * Represents a Person's address in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
  */
-public class Address {
+public class Address implements Printable {
 
     public static final String EXAMPLE = "123, some street";
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String PRINTABLE_ADDRESS_PREFIX = "Address: ";
```
Comment (ZawadulFarhan) [inline src/seedu/addressbook/data/person/Address.java:14]: Good use of static variables!

**Review by ZawadulFarhan — COMMENTED** [review]
Task is well done!

**Pull request closed without merging**